### ISSUE TITLE
BigDebuffs & Tidyplates conflict

### ISSUE THREAD

**Issue by alanrep** [jordonwow/bigdebuffs#122]
Hi,
BigDebuffs is supposed to show on nameplates but it's not working when tidyplates is active. Any way to make them work together ?

**Comment by masonboy93** [comment]
Second this issue. I will gladly cashapp/paypal anyone $20 who can find a way to make these two addons compatible.

**Comment by Theandrin** [comment]
Would love this to work as well, any update? Are any other nameplate addons compatible with BigDebuffs on nameplates? Thanks!

**Comment by jsmenzies** [comment]
@Theandrin Yes I think most of them work as it has support for KUI, plater, threatplates and some others. I'm hoping that this is updated too or else I'm going to have to learn LUA...
